js/cards/disk2.ts

```diff
@@ -842,9 +842,9 @@ export default class DiskII implements Card<State>, MassStorage<NibbleFormat> {
         };
     }
 
-    // TODO(flan): Does not work on WOZ disks
-    rwts(drive: DriveNumber, track: byte, sector: byte) {
-        const curDisk = this.disks[drive];
+    /** Reads the given track and physical sector. */
+    rwts(disk: DriveNumber, track: byte, sector: byte) {
```

Comment:
Not for this PR, but I feel like we'd save ourselves some confusion if we settled on something like `driveNo` for these parameters. I'm not sure why I've avoided doing that up to this point.

Reply:
I quite agree. After the next PR, I'm planning to go through and do a bunch of just documentation and re-naming changes.

Reply:
Here's a commit that just does the drive number renameing: https://github.com/whscullin/apple2js/commit/4402e009dccdc33f19ec9587e4eba4cd40c2c8c4. I'll send this out after the splitting CL. There will probably be some more cleanup, too.